pilot/pkg/serviceregistry/kube/controller/controller.go

```diff
@@ -1005,6 +1004,12 @@ func (c *Controller) getProxyServiceInstancesFromMetadata(proxy *model.Proxy) ([
 		for tp, svcPort := range tps {
 			// consider multiple IP scenarios
 			for _, ip := range proxy.IPAddresses {
+				var network string
+				if proxy.Metadata.Network != "" {
+					network = proxy.Metadata.Network
```

Comment:
In that case, maybe IP ranges take the highest precedence? Most users won't set those and if they do, they have good reason. So it would be 

ipRange > meta > infer from cluster?

Reply:
I remember multi nics requirement is from @zhaohuabing